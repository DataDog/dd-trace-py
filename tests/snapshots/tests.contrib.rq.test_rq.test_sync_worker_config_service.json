[[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "tests.contrib.rq.jobs.job_add1",
      "job.id": "97f1d8cc-a944-48c9-a71f-2f1f316d0a19",
      "language": "python",
      "queue.name": "q",
      "runtime-id": "696c3bcd1bb346e982f3c710445cbea1",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 2601
    },
    "duration": 2230500,
    "start": 1692649888021325467
  },
     {
       "name": "rq.worker.perform_job",
       "service": "my-worker-svc",
       "resource": "tests.contrib.rq.jobs.job_add1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "worker",
       "meta": {
         "_dd.base_service": "",
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "97f1d8cc-a944-48c9-a71f-2f1f316d0a19",
         "job.origin": "q",
         "job.status": "finished",
         "language": "python",
         "runtime-id": "696c3bcd1bb346e982f3c710445cbea1",
         "span.kind": "consumer",
         "traceparent": "00-0000000000000000c7e95eafa9909299-8b78ee45ba9b710e-01",
         "tracestate": "dd=s:1;t.dm:-0"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 2601
       },
       "duration": 3876000,
       "start": 1692649888029838550
     },
        {
          "name": "rq.job.perform",
          "service": "my-worker-svc",
          "resource": "tests.contrib.rq.jobs.job_add1",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "meta": {
            "_dd.base_service": "",
            "component": "rq",
            "job.id": "97f1d8cc-a944-48c9-a71f-2f1f316d0a19"
          },
          "duration": 593958,
          "start": 1692649888030888925
        }]]
