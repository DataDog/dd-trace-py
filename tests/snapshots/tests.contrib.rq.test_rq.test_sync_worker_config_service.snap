[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1630333148705299400
   "duration" 7544600
   "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
           "queue.name" "q"
           "job.id" "db432bb1-6acd-494d-a01f-b3ce14c2c2fb"
           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 462
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.worker.perform_job"
       "service" "my-worker-svc"
       "resource" "tests.contrib.rq.jobs.job_add1"
       "type" "worker"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1630333148723234900
       "duration" 7035700
       "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
               "job.id" "db432bb1-6acd-494d-a01f-b3ce14c2c2fb"
               "job.status" "finished"
               "job.origin" "q"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 462
                  "_dd.tracer_kr" 1.0}}
          {"name" "rq.job.perform"
           "service" "my-worker-svc"
           "resource" "tests.contrib.rq.jobs.job_add1"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1630333148725016000
           "duration" 1024100
           "meta" {"job.id" "db432bb1-6acd-494d-a01f-b3ce14c2c2fb"}}]]
